server/hyrise-includes/ColumnType.h

```diff
@@ -0,0 +1,18 @@
+#ifndef __COLUMNTYPE_H_
```

Comment:
let's try to remove other dependencies on hyrise code except the parser syntax file part

Reply:
I will remove them.

Reply:
done